solidity/contracts/bridge/Bridge.sol

```diff
@@ -1944,4 +1946,47 @@ contract Bridge is
     function txProofDifficultyFactor() external view returns (uint256) {
         return self.txProofDifficultyFactor;
     }
+
+    /// @notice Sets the redemption watchtower address.
+    /// @param redemptionWatchtower Address of the redemption watchtower.
+    /// @dev Requirements:
+    ///      - The caller must be the governance,
+    ///      - Redemption watchtower address must not be already set,
+    ///      - Redemption watchtower address must not be 0x0.
+    function setRedemptionWatchtower(address redemptionWatchtower)
+        external
+        onlyGovernance
+    {
+        self.setRedemptionWatchtower(redemptionWatchtower);
```

Comment:
for those of us not super familiar with the codebase, it would be handy to have a comment that highlights that this function is defined in `BridgeState.sol`

Reply:
Good point! Please see 743e9c75